opal/mca/threads/pthreads/threads_pthreads_wait_sync.c

```diff
@@ -7,6 +7,7 @@
  *                         reserved.
  * Copyright (c) 2017      IBM Corporation. All rights reserved.
  * Copyright (c) 2019      Sandia National Laboratories.  All rights reserved.
+ * Copyright (c) 2021      Argonne National Laboratory.  All rights reserved.
```

Comment:
Why adding the copyright header here? I don't see any changes.

Reply:
You're right. I should have changed the `threads_argobots_...*.c`. I fixed it.